docs/docs/release.md

```diff
@@ -0,0 +1,36 @@
+# Release
+
+This section covers miscellaneous tasks to do with the release.
+
+## ptxcompiler and cubinlinker
+
+In addition to the standard RAPIDS packages, RAPIDS also published `ptxcompiler` and `cubinlinker` Python packages.
+These are CUDA 11-specific tools to enable [MVC](https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#cuda-compatibility-developer-s-guide) for JIT-compiled code using numba.
```

Comment:
"MVC" is not on this linked page. Can you define the acronym? Maybe it has a new name, or has changed location?